hardware/reference-design.md

```diff
@@ -26,8 +26,8 @@ Because the micro:bit has a Bluetooth radio, building your own boards just like
 the micro:bit is complicated - you need to tune the antenna, add the right
 resonance circuit and then certify the board, if you'd like to build more than
 just a few of them. Therefore, in order to make it easier to build things based
-on the micro:bit, we've made a version of the board that uses an nRF51-based
-module, so that you can just worry about adding your own components to the board.
+on the micro:bit, we've made this version of the board that uses a pre-certified nRF51-based
+module (details in [Module Choice](#module-choice) below). This way you can leave your mind off the gritty RF details and just need to worry about adding your own components to the board.
```

Comment:
leave your mind off the gritty RF details and just need to worry about adding your own components to the board.
> take your mind off the gritty RF details and only need to worry about adding your own components to the board.